api/python/provisioner/commands/auto_deploy.py

```diff
@@ -50,6 +50,7 @@ class AutoDeploy(SetupCmdBase, CommandParserFillerMixin):
         RunArgsSetupProvisionerGeneric,
         deploy_dual.run_args_type
     ]
+    _description = 'API to deploy Cortx on Hardware'
```

Comment:
I think underscore is not necessary here since we are going to access that variable outside the class/children

Reply:
comment is addressed